app/routes/event/$eventId.tsx

```diff
@@ -40,13 +40,14 @@ export const loader: LoaderFunction = async ({ request }) => {
   }
 };
 
+const shouldRevalidate: ShouldRevalidateFunction = ({}) => false;
+
```

Comment:
What is this doing?

Reply:
We are currently working on conditionally render the events or load the events based on the user actions. The above book is built in tool by remix which would help us
@isVivek99 would be taking this up to work further to reduce API calls for improving performance